ogticrd/covid-safe-paths
# Storage encryption

#### Description:
As a user, I want the data in my phone storage encrypted so that data keeps safe even when is saved in the phone.


#### Linked issues:
[Ticket 235](https://trello.com/c/QIMPMnwo/235-as-a-user-i-want-the-data-in-my-phone-storage-encryted-so-that-data-keeps-safe-even-when-is-saved-in-the-phone-1)
[Ticket 262](https://trello.com/c/zBhUceWe/262-use-for-others-in-positive-is-not-working)


#### Screenshots:



#### How to test:
`git fetch && git checkout storage-encryption` place a console log in app/helpers/General.js/GetStorageData, below `let data = await AsyncStorage.getItem(key);` and below `data = data !== null ? Decrypt(data) : data;` and see the change between encrypted and decrypted.
